posts/views.py

```diff
@@ -39,6 +39,15 @@ def list(self, request) -> Response:
 class LikeDislikeViewSet(viewsets.ModelViewSet):
     permission_classes = (IsAuthenticated,)
 
+    def get(self, request, pk: int):
```

Comment:
What is the URL for this endpoint?

Reply:
both http://127.0.0.1:8000/api/posts/1/like/ and http://127.0.0.1:8000/api/posts/6/dislike/

Reply:
Let's have something like
`/api/posts/{id}/likes_count/` or `/api/posts/{id}/likes/count/` or `/api/posts/{id}/likes/stats/` or `/api/posts/{id}/likes/`

Reply:
Done, now it's `/api/posts/{id}/likes/count/`